types/index.d.ts

```diff
@@ -27,7 +27,7 @@ declare namespace remarkExternalLinks {
      *
      * @defaultValue ['nofollow', 'noopener', 'noreferrer']
      */
-    rel?: string[] | string
+    rel?: string[] | string | boolean
```

Comment:
looking at how this is handled by the javascript code, I think `false` is supported, but `true` is not:
https://github.com/remarkjs/remark-external-links/blob/2c6fc4bd9801489809789964dff96c9ecf6e0605/index.js#L54-L56

```suggestion
    rel?: string[] | string | false
```